packages/components/src/components/autocomplete/autocomplete.jsx

```diff
@@ -68,7 +68,7 @@ const Autocomplete = React.memo(props => {
             setActiveIndex(null);
             setInputValue('');
         }
-    }, [list_items]);
+    }, [list_items, has_updating_list]);
```

Comment:
please add this component to qa check list

Reply:
Yes, I'll add this [packages/components/src/components/vertical-tab/vertical-tab-headers.jsx](https://github.com/binary-com/deriv-app/pull/5943/files/3b0a6fcb43b0a852b0c15548d49cb0569fa10c9d#diff-3658191ed864a4c75732cb1c6472ce89199b66b9335dec6ff4869bb24a855fcb) as well